cypress/integration/utils/devices.ts

```diff
@@ -80,6 +107,18 @@ function getAppLayoutTests(tests: Tests, skipNavbar = false, loggedIn = false) {
   return [...testSignedOutNavbar, ...tests];
 }
 
+function getAppLayoutTestsMobile(
+  tests: Tests,
+  skipNavbar = false,
+  loggedIn = false,
+) {
+  if (skipNavbar) return [...tests];
```

Comment:
Don't need to destructure `tests` here
```suggestion
  if (skipNavbar) return tests;
```